.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Report a run-time problem
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Expected vs observed behaviour**
+Please describe what you expected to happen and what actually happens.
+
+**Patch or snippet allowing to reproduce the problem**
+
+Please attach the smallest possible patch allowing to reproduce the problem in a sample. If there is absolutely no appropriate sample, please attach a minimal self-contained example in a single file.
+
+Skip this step if the problem can be reproduced in one of the samples without any changes.
+
+**To Reproduce**
+Steps to reproduce the behaviour, please make them as detailed as possible, e.g.
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Platform and version information**
+ - wxWidgets version you use: [e.g. 3.1.6]
+ - wxWidgets port you use: [e.g. wxMSW, wxGTK, wxOSX]
+ - OS and its version: [e.g. Windows 10.0.19042.867, Ubuntu 22.10, macOS 15]
```

Comment:
I have no knowledge of Linux and I understand that this is supposed to be brief but I wonder if we should ask also for GTK version and desktop environment (and theme used, if non-default?) there...

Reply:
This is a good point, we do want to have GTK version and the theme can be useful too, thanks! Will add.